File: PHP Word Examples/SectionsApiDemo/getSections.php
<?php

use Spire\Cloud\Word\Sdk\Client\Configuration;
use Spire\Cloud\Word\Sdk\Client\Api\SectionsApi;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new SectionsApi($configuration);

$name = "PageSetup.docx";
$folder = "input";
$storage =null;
$password =null;
$result = $apiInstance->getSections($name, $folder, $storage, $password);

File: Python Word Examples/SectionsApiDemo/getSections.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.SectionsApi(configuration)

file_name = 'PageSetup.docx'
extra = {'folder': "input"}
result = api.get_sections(file_name, **extra)